logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/EventRepository.kt

```diff
@@ -115,6 +115,17 @@ class EventDataSource(
                     }
                 )
             }
+
+            is EventDeliveryInfo.AsyncMissed -> {
+                currentClientId().fold(
+                    { it.left() },
+                    {
+                        // todo(ym) check for errors.
```

Comment:
TODO

Reply:
I will address all error handling in a separate PR, as it is related to a wider task